styles/main.css

```diff
@@ -49,51 +50,62 @@ nav ul {
 }
 
 footer {
-  text-align: center;
-  font-size: 12px;
-  padding: 0 0 2em 0;
-  margin-bottom: 2em;
+    text-align: center;
+    font-size: 12px;
+    padding: 0 0 2em 0;
+    margin-bottom: 2em;
 }
 
 ul {
     list-style-type: upper-roman;
-    }
-#width-wrap {
+}
+
+.width-wrap {
     margin-left: auto;
     margin-right: auto;
     max-width: 1024px;
     background-color: #D833FF;
     box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.2);
     padding: 0 2em 0 2em;
     margin-bottom: 3em;
-    border: 1px solid rgb(0 0 0 / 0.3);
+    border: 1px solid rgba(0, 0, 0, 0.3);
+    border-top: none;
     
 }
-header nav ul{
+
+header nav ul {
     padding: 0;
-} 
+    margin: 0;
+}
 
 
-figure img{
+figure img {
     box-shadow: 2px 2px 3px rgba(0, 0, 0, 0.5);
-} 
-
-h, h1, h2, h3, h4, h5, h6 {
+}
 
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
     text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.2);
-    background: rgb(2,0,36);
-    background: linear-gradient(180deg, rgba(2,0,36,1) 0%, rgba(121,9,117,1) 55%, rgba(255,0,249,1) 100%);
-    -webkit-text-fill-color: transparent;
+    background: rgb(2, 0, 36);
+    background: linear-gradient(180deg, rgba(2, 0, 36, 1) 0%, rgba(121, 9, 117, 1) 55%, rgba(255, 0, 249, 1) 100%);
+    color: transparent;
     -webkit-background-clip: text;
+    background-clip: text;

```

Comment:
If possible, always include the correct property in addition to vendor-prefixed properties